init.sql

```diff
@@ -1,79 +1,76 @@
-CREATE TABLE Account (
-	accountID varchar(20) SERIAL PRIMARY KEY, /* Serial does automatic increment? https://chartio.com/resources/tutorials/how-to-define-an-auto-increment-primary-key-in-postgresql/ */
-	password varchar(50) NOT NULL
-);
-
-CREATE TABLE User (
+CREATE TABLE AppUser (
 	username varchar(50) PRIMARY KEY,
+	password varchar(50) NOT NULL,
     firstName varchar(20) NOT NULL,
     lastName varchar(20) NOT NULL,
     address varchar(50) NOT NULL,
-    password varchar(50) NOT NULL,
-    phoneNumber integer NOT NULL
+    phoneNumber varchar(20) NOT NULL
 );
 
 CREATE TABLE Driver (
-	username varchar(50) PRIMARY KEY REFERENCES User ON DELETE cascade,
+	username varchar(50) PRIMARY KEY REFERENCES AppUser ON DELETE cascade,
 	d_rating integer,
 	license_no integer NOT NULL
 );
 
 CREATE TABLE Passenger (
-	username varchar(50) PRIMARY KEY REFERENCES User ON DELETE cascade,
+	username varchar(50) PRIMARY KEY REFERENCES AppUser ON DELETE cascade,
 	p_rating integer
 );
 
 CREATE TABLE Model (
-	name varchar(20) PRIMARY KEY,
-    size integer
+	name varchar(20),
+	brand varchar(50),
+    size INTEGER NOT NULL,
+	PRIMARY KEY (name, brand)
 );
 
 CREATE TABLE Car (
 	plateNumber varchar(20) PRIMARY KEY,
-    colours  varchar(20)
+    colours  varchar(20) NOT NULL
 );
 
 CREATE TABLE Promo (
 	promoCode varchar(20) PRIMARY KEY,
-	quotaLeft integer,
-	maxDiscount integer,
-	minPrice integer,
-	discount integer	
+	quotaLeft integer NOT NULL,
+	maxDiscount integer,   -- Can we put default here?
+	minPrice integer,		
+	discount integer NOT NULL	
 );
 
 CREATE TABLE Ride (
-	rideID varchar(20) PRIMARY KEY,
+	rideID varchar(20) PRIMARY KEY,	-- Is this auto increment?
     p_comment varchar(50),
     p_rating integer,
     d_comment varchar(50),
     d_rating integer	
 );
 
 CREATE TABLE Advertisement (
-    	timePosted integer PRIMARY KEY,
+	timePosted integer PRIMARY KEY,	-- Change to AdID instead? Since this is a driverID concat with time
     numPassengers integer NOT NULL,
     departureTime integer NOT NULL,
     price integer NOT NULL,
-    to varchar(50) NOT NULL,
-    from varchar(50) NOT NULL
+    toPlace varchar(50) NOT NULL,
+    fromPlace varchar(50) NOT NULL
```

Comment:
To use pre-defined places by creating a new entity called `place` where users will select from the drop down box.